packages/core/src/common/set-zero-time-out.ts

```diff
@@ -0,0 +1,16 @@
+export function setZeroTimeout(callback: any): () => void {
```

Comment:
建议叫 `requestImmediateMacroTask`

另外回调参数类型尽量不要用

```suggestion
export function requestImmediateMacroTask(callback: () => void): () => void {
```

Reply:
参数用()=>void会报类型错误

Reply:
(...args: any[]) => void 也不行么